modules/transactions/src/main/java/org/apache/ignite/internal/tx/impl/HeapLockManager.java

```diff
@@ -251,22 +251,18 @@ public void releaseAll(UUID txId) {
         }
     }
 
+    @Override
+    public Iterator<Lock> locks() {
```

Comment:
why do we need additional method if we can gather all txId and further call locks(UUID txId) for each ?

Reply:
got it, let\`s call also @vldpyatkov here